### PR TITLE
Fix homepage hero

### LINKED ISSUE #96: Hero section does not render as expected
The hero section of the deployed website happens to be slightly different from what's on the design.

**Current**
Current

**Expected**
Expected

### PR DESCRIPTION
**Pull request by iamuchejude**
Fixes #96

### CODE REVIEW

**Comment by frontibotito** [comment]
Deploy preview for _website_ ready!

Built with commit c6fbb0d8130e0057c980b9aae153d48d597b3926

https://frontity-org-pb0o9jzao.now.sh

File: packages/frontity-org-theme/src/components/styles/homepage.tsx [hunk 1]
```diff
@@ -38,23 +39,14 @@ const homePageStyles = (state: FrontityOrg["state"]["theme"]) => css`
...
-      right: 0;
-      top: 0;
-      height: 100%;
-      width: 100%;
+    clip-path: polygon(0 0, 100% 0, 100% 88%, 0% 100%);
 
-      clip-path: polygon(0 100%, 100% 85%, 100% 92%);
+    @media only screen and (max-width: 769px) {
+      height: auto;
+      clip-path: none;
```
Comment (DAreRodz) [inline packages/frontity-org-theme/src/components/styles/homepage.tsx:49]: Hey, you removed the `clip-path` for the mobile version. Maybe a different `clip-path` is needed but it should have one, according to the design.

File: packages/frontity-org-theme/src/components/homepage-hero.tsx [hunk 2]
```diff
@@ -36,9 +36,14 @@ const Container = styled.div`
   height: 100%;
 
   & > div {
+    clip-path: polygon(0 0, 300% 0, 120% 90%, 0% 100%);
     position: relative;
     height: 100%;
 
+    @media only screen and (max-width: 769px) {
+      clip-path: none;
```
Comment (DAreRodz) [inline packages/frontity-org-theme/src/components/homepage-hero.tsx:44]: Same here.

**Comment by SantosGuillamot** [comment]
Closing this as it doesn't seem to apply anymore.

**Pull request closed without merging**